mcp/cortex_mcp_server.py

```
@@ -0,0 +1,231 @@
+        result = await self._run_cortex(args)
+        return {"installation_id": installation_id, "mode": "dry_run" if dry_run else "execute", **result}
+
+    async def _detect_hardware(self) -> dict:
+        hardware = {}
+        try:
+            with open("/proc/cpuinfo") as f:
+                for line in f:
+                    if line.startswith("model name"):
+                        hardware["cpu"] = line.split(":")[1].strip()
+                        break
+        except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)